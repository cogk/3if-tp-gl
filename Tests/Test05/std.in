2
1
424242.42
60.123456
50.001
3
