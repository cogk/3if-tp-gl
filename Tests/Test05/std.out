* Bienvenue dans AirWatcher 
| Utilisateur: (anonyme)       
* Menu Principal 

   1. Contribution…
   2. Analyse…
   3. Sources de données…
   4. Administration…
   5. Super Admin…
   6. Quitter AirWatcher

Entrez ici votre sélection : 
* Menu - Analyse 

   1. Filtrage géographique…
   2. Filtrage temporel…
   3. Lancer le calcul
   4. Retour au menu principal

Entrez ici votre sélection : 
* Définir une région d'analyse 
Forme de la région d'analyse : CERCLE
* Rayon du cercle (en mètres) : * Centre du cercle
   - latitude :    - longitude : 
>> Filtre géographique : Cercle de rayon 424242 et de centre (60.1235, 50.001)
* Menu - Analyse 

   1. Filtrage géographique…
   2. Filtrage temporel…
   3. Lancer le calcul
   4. Retour au menu principal

Entrez ici votre sélection : 
Calcul...* Aucun résultat pour les critères spécifiés 

>> Filtre géographique : Cercle de rayon 424242 et de centre (60.1235, 50.001)
* Menu - Analyse 

   1. Filtrage géographique…
   2. Filtrage temporel…
   3. Lancer le calcul
   4. Retour au menu principal

Entrez ici votre sélection : 
Entrée invalide, veuillez entrer un entier entre 1 et 4.
Vous n'avez pas choisi de menu.
* Menu Principal 

   1. Contribution…
   2. Analyse…
   3. Sources de données…
   4. Administration…
   5. Super Admin…
   6. Quitter AirWatcher

Entrez ici votre sélection : 
Entrée invalide, veuillez entrer un entier entre 1 et 6.
Vous n'avez pas choisi de menu.
